### PR TITLE
Auto adjust scrollable dialog's height based on content

### PR DESCRIPTION
**Pull request by d3radicated**
Currently, on scrollable dialogs, you need to set `v-card-text`'s height for it to properly show its contents. Also, where `v-card-text`'s height is too high, it will hide `v-card-actions`. This PR will auto adjust `v-card-text`'s height based on content and it will not hide `v-card-actions`.

### CODE REVIEW

**Comment by johnleider** [comment]
Please provide an example markup that will exhibit this behavior without your fix, thank you.

**Comment by d3radicated** [comment]
From docs page: https://codepen.io/dEradicated/pen/OxWzeJ 
With fix: https://codepen.io/dEradicated/pen/MEbQar

Fiddle with the screen's height to see the behavior.

**Comment by johnleider** [comment]
Looks good, only issue I see is the ```v-divider```'s are no longer working with your fix.

**Comment by d3radicated** [comment]
I'll see what I can do with that. 😉

**Comment by KaelWD** [comment]
`.divider` should have `flex: 0 0 1px`

File: src/stylus/components/_dialogs.styl [hunk 1]
```diff
@@ -46,8 +46,19 @@
...
-
-    .card__text
-      overflow-y: auto
-      backface-visibility: hidden
+    
+    >
+      .card
+        display: flex
+        flex: 1 1 auto
+        flex-flow: column
```
Comment (KaelWD) [inline src/stylus/components/_dialogs.styl]: This can just be `flex-direction`

Comment (d3radicated) [reply src/stylus/components/_dialogs.styl]: Updated.

**Comment by d3radicated** [comment]
I changed `v-divider`'s style by setting its `min-height` instead of just `height`;

**Comment by KaelWD** [comment]
Put it back to `height` and change `flex` instead. Dividers should never be anything other than 1px high, so they shouldn't have `flex-grow` and `flex-shrink` like they do now.

**Comment by d3radicated** [comment]
I changed it to `min-height` because changing it to `flex: 0 0` will break the steppers.

**Comment by KaelWD** [comment]
Oh yeah they can be horizontal as well...  
You should still keep `height` at least because otherwise they'll be too tall in a fixed-height container.

**Comment by d3radicated** [comment]
Done and done.

**Pull request merged**